### DISCUSSION TITLE
Enterprise portal | Not able to publish documents

### DISCUSSION THREAD

**Discussion by ghost** [pimcore/pimcore#11925] [Q&A / Support]
Hi,

I have setup the pimcore enterprise and now I am facing the error while creating the document for front-end portal.

![image](https://user-images.githubusercontent.com/52573912/164004259-d32b87fb-1255-414f-a78b-0bd94b1d7e7f.png)

Pls, help if I miss some configuration.

**Comment by fashxp** [comment]
please contact your pimcore contact to get further support... thx

**Reply by ghost** [reply]
Thanks, Christian for your reply.
